packages/flutter/lib/src/material/autocomplete.dart

```diff
@@ -192,6 +193,11 @@ class Autocomplete<T extends Object> extends StatelessWidget {
   /// default.
   final AutocompleteOptionsViewBuilder<T>? optionsViewBuilder;
 
+  /// The initial value to use for the text field.
+  ///
+  /// This parameter is ignored if [textEditingController] is defined.
```

Comment:
nit: maybe use a macro for this?

Reply:
It looks like setting the initial value does not notify `textEditingController`'s listeners, so the dropdown menu won't show (maybe worth documenting?)